controllers/clustergroupupgrade_controller.go

```diff
@@ -1125,13 +1174,16 @@ func (r *ClusterGroupUpgradeReconciler) blockingCRsNotCompleted(ctx context.Cont
 			continue
 		}
 
-		// If we find a blocking CR that does not contain a true succeeded condition then we add it to the list.
-		if !meta.IsStatusConditionTrue(cgu.Status.Conditions, string(utils.ConditionTypes.Succeeded)) {
+		if partialComplete == true {
+			if meta.IsStatusConditionTrue(cgu.Status.Conditions, string(utils.ConditionTypes.Progressing)) {
```

Comment:
Progressing can be false before the CGU starts. I thought we had to use the existence of the succeeded condition.

Reply:
how about progressing=false, succeded=false

Reply:
updated